.github/workflows/compile-artifact.yml

```diff
@@ -35,7 +41,8 @@ jobs:
       - name: Configure bundler
         run: |
           mkdir -p vendor/bundle
-          bundle config path vendor/bundle
+          bundle config set --local path vendor/bundle
+          bundle config set --local gemfile Gemfile${{ inputs.gemfile_suffix }}
```

Comment:
running bundle config will create a persistent config file in .bundle with these settings
is this intended? was there a reason for going this way instead of using the bundler env vars?

I am legit not sure if there will be a conflict/trampling situation between our own bundle config file and elasticbeanstalks.  EB does run these commands so maybe they get appended just fine?
```
/bin/sh -c bundle config set --local deployment true
/bin/sh -c bundle install --local
```

Reply:
This is actually just copied from the Buddy -- if there's a more compliant way, snippet me / link me up!

Reply:
After a Zoom call, we determined that the local variables are the highest precedence and Buddy was set to use those, so it's best to continue with what we had (and with what also has most precedence)